### PR TITLE
Create MAINTENANCE.md

### PR DESCRIPTION
**Pull request by SomeHats**
i wrote this like half a week ago and forgot to pr it oops

Much of this is adapted from http://sous-chefs.org/, but with most of the details left out. We should flesh out the 'What does day-to-day maintenance by fruitful.js look like' section in particular. Each of those bullet points wants to be a section heading, but we need to decide how they're gonna work

✨

### CODE REVIEW

File: MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+When a project is first transferred to fruitful.js it will:
+
+- Adopt our code of conduct and enforcement policy
+- Move to the fruitfuljs GitHub organisation
+- Get listed on the fruitful.js project website.
+
+You can read more about the transfer process on the [migrating](MIGRATING.md)
+page.
+
+### What does day-to-day maintenance by fruitful.js look like?
```
Comment (pup) [inline MAINTENANCE.md]: not sure if `by` is the right word here? feels like the org itself is doing all this, and not the people. (i can't think of a better alternative though)

File: MAINTENANCE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Managing an issue tracker
```
Comment (pup) [inline MAINTENANCE.md]: instead of `an issue tracker`, maybe say `the issue tracker`?

File: MAINTENANCE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Keeping dependencies up to date (with the help of https://greenkeeper.io/)
+- Fixing reported bugs & security issues
+- Responding to feature requests
+- Improving and updating documentation and automated tests
+
+### Stopping maintenance
+If a project in unused or we're no longer able to support it, we may decide to
```
Comment (pup) [inline MAINTENANCE.md]: `in` -> `is`

File: MAINTENANCE.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+stop maintaining it. If this happens, we'll clearly mark the project as
+unmaintained or deprecated in it's README.
```
Comment (pup) [inline MAINTENANCE.md]: `it's` -> `its`

**Comment by pup** [comment]
(oh yeah, also fruitful should consistently be referred to as `Fruitful`, not `fruitful.js`, `fruitfuljs` or anything like that :smile:)

File: MAINTENANCE.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Maintenance
+
+### What happens when a project is first transferred to fruitful.js?
+When a project is first transferred to fruitful.js it will:
+
+- Adopt our code of conduct and enforcement policy
+- Move to the fruitfuljs GitHub organisation
+- Get listed on the fruitful.js project website.
```
Comment (igalic) [inline MAINTENANCE.md]: once in shape, a new release should be made in its old namespace, that deprecates it, and points to the new namespace.

File: MAINTENANCE.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+You can read more about the transfer process on the [migrating](MIGRATING.md)
+page.
+
+### What does day-to-day maintenance by fruitful.js look like?
+
+- Managing an issue tracker
+- Keeping dependencies up to date (with the help of https://greenkeeper.io/)
+- Fixing reported bugs & security issues
+- Responding to feature requests
+- Improving and updating documentation and automated tests
```
Comment (igalic) [inline MAINTENANCE.md]: aren't tests pretty much the main requirement for greenkeeper?

File: MAINTENANCE.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+### Stopping maintenance
+If a project in unused or we're no longer able to support it, we may decide to
+stop maintaining it. If this happens, we'll clearly mark the project as
+unmaintained or deprecated in it's README.
+
+If someone outside the fruitful.js project wants to continue maintaining the
+project and forks it, we can update the README to point to the maintained fork.
```
Comment (igalic) [inline MAINTENANCE.md]: hrm… why are we deciding to drop maintenance?

when someone is found who wants to maintain the project, could we just give them org access?

File: MAINTENANCE.md (hunk 6 above)
Comment (SomeHats) [inline MAINTENANCE.md]: yup. I was thinking of adding something to the 'when a project is first transferred' bit about cleaning up the project and adding all that sort of thing.

We should figure out a set of preferred tools like http://sous-chefs.org/contributing/#cookbook-testing does though. I figured going into more detail in these docs could happen later.

File: MAINTENANCE.md (hunk 7 above)
Comment (SomeHats) [inline MAINTENANCE.md]: This is from http://sous-chefs.org/departure/#when-the-maintainer-wants-to-leave. 

I guess we could have a case where someone wants to maintain a project but doesn't want to be part of fruitful? I can just drop the line though.

**Comment by SomeHats** [comment]
I'd say we could start with keeping them as subsections in this file, but if things get too long we can split the document in two?

**Pull request merged**